src/langcheck/metrics/en/reference_based_text_quality.py

```diff
@@ -70,26 +80,40 @@ def semantic_similarity(
         model = SentenceTransformer('sentence-transformers/all-mpnet-base-v2')
         generated_embeddings = model.encode(generated_outputs)
         reference_embeddings = model.encode(reference_outputs)
-    else:  # openai
+    else:  # openai or azure_openai
+        if not openai_client:
```

Comment:
Adding the same comment as in Koki's rephrasing PR: https://github.com/citadel-ai/langcheck/pull/56
```suggestion
        # Initialize the openai object if openai_client is None
	    # TODO: Refactor this into OpenAIBasedEvaluator?
        if not openai_client:
```